beets/library.py

```diff
@@ -247,6 +247,21 @@ def add(self, lib=None):
         super(LibModel, self).add(lib)
         plugins.send('database_change', lib=self._db)
 
+    def __format__(self, spec):
+        if not spec:
+            spec = beets.config[self._format_config_key].get(unicode)
+        result = self.evaluate_template(spec)
+        if isinstance(spec, bytes):
```

Comment:
Ah, I get it—you're supporting formatting with both bytes and Unicode templates. Sorry for being slow. Perhaps a comment to this effect would make it clearer?